addons/point_of_sale/static/tests/tours/PaymentScreen.tour.js

```diff
@@ -290,13 +290,14 @@ registry.category("web_tour.tours").add("CashRoundingPayment", {
             PaymentScreen.totalIs("1.90"),
             PaymentScreen.clickPaymentMethod("Cash"),
             PaymentScreen.pressNumpad("1 ."),
-            PaymentScreen.pressNumpad("2 4"),
-            PaymentScreen.fillPaymentLineAmountMobile("Cash", "1.24"),
-            PaymentScreen.selectedPaymentlineHas("Cash", "1.24"),
+            PaymentScreen.pressNumpad("9 4"),
+            PaymentScreen.fillPaymentLineAmountMobile("Cash", "1.94"),
+            PaymentScreen.selectedPaymentlineHas("Cash", "1.94"),
```

Comment:
why do you need this change ?

Reply:
To be able to click on validate, the remaining must be zero